README.md

```diff
@@ -18,7 +18,7 @@ This will be the best place to discuss design and implementation.
 
 For sync communication we have a community slack with a #containerd channel that everyone is welcome to join and chat about development.
 
-**Slack:** https://community.docker.com/registrations/groups/4316
+**Slack:** http://dockr.ly/MeetUp
```

Comment:
The community Slack channels are for a range of topics including containerd. Victor created the link because the captains could never remember the URL to share. This is an improvement on previous link.